airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingDestination.java

```diff
@@ -165,7 +165,7 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
           new DefaultTyperDeduper<>(sqlGenerator, snowflakeDestinationHandler, parsedCatalog, migrator, v2TableMigrator, defaultThreadCount);
     }
 
-    return new StagingConsumerFactory().createAsync(
+    return new StagingConsumerFactory.Builder(
```

Comment:
nit: Factory pattern is to expose instance creation using a static method. `Factory.builder()...build()`

Reply:
done in https://github.com/airbytehq/airbyte/pull/34502/commits/52a10a3959434f73b716f00a11342d135b6e36fb